react/components/Pagination/index.js

```diff
@@ -60,49 +63,64 @@ class Pagination extends PureComponent {
     const itemTo = currentItemTo > totalItems ? totalItems : currentItemTo
 
     return (
-      <div
-        className={`flex flex-row items-center ${
-          rowsOptions ? 'justify-between' : 'justify-end'
-        }`}>
-        {dropdownOptions && (
-          <div className="flex flex-row pt5 items-baseline">
-            <span className="mr4 c-muted-2 t-small self-center">
-              {textShowRows}
-            </span>
-            <Dropdown
-              size="small"
-              options={dropdownOptions}
-              value={
-                selectedOption || dropdownOptions[selectedRowsOptionIndex].label
-              }
-              onChange={this.handleRowsChange}
+      <div className="flex flex-column">
+        {hasPageTopIndicator && (
+          <div className="mt2 mb5">
+            <PageIndicator
+              currentItemFrom={currentItemFrom}
+              itemTo={itemTo}
+              textOf={textOf}
+              totalItems={totalItems}
             />
           </div>
         )}
-
-        <div className="flex flex-row pt5 items-center">
-          <div className="c-muted-2 t-small">
-            {currentItemFrom}
-            {' - '}
-            {itemTo} {textOf} {totalItems}
-          </div>
-          <div className="ml4">
-            <ButtonWithIcon
-              icon={caretLeft}
-              variation="secondary"
-              size="small"
-              disabled={isPrevDisabled}
-              onClick={this.handlePrevPage}
-            />
-          </div>
-          <div className="ml2">
-            <ButtonWithIcon
-              icon={caretRight}
-              variation="secondary"
-              size="small"
-              disabled={isNextDisabled}
-              onClick={this.handleNextPage}
+        {this.props.children}
+        <div
+          className={`flex flex-row items-center ${
+            rowsOptions ? 'justify-between' : 'justify-end'
+          }`}>
```

Comment:
why do you not use `classnames` function?

Reply:
Done, although there is just one flag so far, for the sake of expandability.